### DISCUSSION TITLE
The nats-server  v2.1.2 is used by kit , I noticed nats-server v2.1.2 has a High-Risk Vulnerability CVE-2022-24450, However is kit affetcd by this CVE ?

### DISCUSSION THREAD

**Discussion by liufangwai** [go-kit/kit#1223] [Q&A] [answered]
The nats-server  v2.1.2 is used by kit , I noticed nats-server v2.1.2 has a High-Risk Vulnerability CVE-2022-24450, However is kit affetcd by this CVE ?

**Comment by ChrisHines** [comment, marked as answer]
No. The most recent release of go-kit (v0.12.0) uses nats-server v2.5.0 and **only for tests**. The non-test code in go-kit does not use nats-server at all, only the nats client.

**Reply by liufangwai** [reply]
ok,  thanks for you apply
